docs/topics/optimize-performance.adoc

```diff
@@ -2,6 +2,7 @@
 //
 // * docs/cli-guide/master.adoc
 
+:_content-type: REFERENCE
 [id="optimize-performance_{context}"]
 = Optimizing {ProductShortName} performance
```

Comment:
@sbeskin-redhat @RichardHoch 
1. This is a CONCEPT.
2. I'm on the fence regarding whether this one module should actually be three. If you want to keep it as one, fine.
3. IBM and RH style guidance says to avoid self-referential sentences like "This section describes what you can do to improve the performance of MTA.". I suggest changing it to something like "You can optimize performance by adjusting the manner in which you deploy and run your application, or by upgrading the hardware on which the application runs."
4. The two subheadings suggest procedures, but these are concepts. I suggest changing the titles to reflect that.
This last comment applies to several modules I'm seeing.